WordPress/src/main/java/org/wordpress/android/ui/accounts/signup/SignupMagicLinkFragment.java

```diff
@@ -76,7 +76,9 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         mTryAgainButton.setOnClickListener(new View.OnClickListener() {
```

Comment:
To be super clear, wdyt about renaming `mTryAgainButton` to better match its new functionality?

Reply:
I update the variable in 822a5415b076c0095dbf91368b748aa1dcaeecea.